app/pages/resource/info/ResourceInfo.js

```diff
@@ -1,13 +1,40 @@
 import React, { PropTypes } from 'react';
+import Col from 'react-bootstrap/lib/Col';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+import Row from 'react-bootstrap/lib/Row';
+
+import ImageCarousel from 'shared/image-carousel';
+import WrappedText from 'shared/wrapped-text';
 
 function ResourceInfo({ resource, unit }) {
   return (
     <div className="resource-info">
-      <h1>
-        <span className="unit-name">{unit.name.fi}</span>
-        {' '}
-        <span className="resource-name">{resource.name.fi}</span>
-      </h1>
+      <header>
+        <h2 className="unit-name">{unit.name.fi}</h2>
+        <h1 className="resource-name">{resource.name.fi}</h1>
+        <h4 className="unit-address">
+          <Glyphicon glyph="map-marker" className="map-marker" /> {unit.streetAddress.fi}
```

Comment:
- [ ] The address should also contain `addressZip` and `municipality`. You could add a general unitSelector, which adds the fullAddress to the unit. Note that unit.streetAddress might be `null`.